include/roboteam_utils/Tube.h

```diff
@@ -0,0 +1,58 @@
+//
+// Created by rolf on 24-02-20.
+//
+
+#ifndef ROBOTEAM_UTILS_TUBE_H
+#define ROBOTEAM_UTILS_TUBE_H
+#include "LineSegment.h"
+namespace rtt{
+/**
+ * @author Rolf
+ * @date 24-02-20
+ * @brief A class that represents a tube, which can be seen as the points at a constant distance from a 2d line segment or a capped 2d cylinder
+ */
+class Tube {
+    public:
+        /**
+         * @brief Constructs a default Tube, which is simply a unit circle.
+         */
+        Tube();
+        /**
+         * @brief Construct a tube around the line from start to end with radius
+         * @param start Starting point
+         * @param end  Ending point
+         * @param radius Given radius
+         */
+        Tube(const Vector2& start, const Vector2& end, double radius);
+        /**
+         * @brief Construct a tube around the line with given radius
+         * @param line Line to construct tube around
+         * @param radius Given radius
+         */
+        Tube(const LineSegment& line, double radius);
+
```

Comment:
Can we construct a tube by passing another tube?

Reply:
Yes, but this should be defaulted correctly already. We can be explicit in the header but it won´t make an effective difference.